com.unity.render-pipelines.high-definition/CHANGELOG.md

```diff
@@ -66,6 +66,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Added missing documentation for unsupported SG RT nodes and light's include for raytracing attrbute.
 - Added documentation for LODs not being supported by ray tracing.
 - Added more options to control how the component of motion vectors coming from the camera transform will affect the motion blur with new clamping modes.
+- Added a terrain compatiblity shader tag for validating whether user overridden materials support terrain rendering functionalities. 
```

Comment:
Suggestion:
"Before this version, when using Terrain Toolbox, if a user selected a Material that did not use one of Unity's pre-built Terrain shaders in the Material Override field, Unity showed a warning. This version of the package implements the TerrainCompatible SubShader Tag. Adding this tag to a custom Terrain shader tells Unity that the shader is compatible with the Terrain system, and Unity does not show the warning if a user selects a Material with such shader in the Material Override field."

Reply:
I made a few edits

"Before this version, when ~~using Terrain Toolbox, if~~  a user selected a Material that did not use one of Unity's pre-built Terrain shaders in **a** ~~the~~ Material Override field, Unity **did not** show~~ed~~ a warning. This version of the package implements the TerrainCompatible SubShader Tag. Adding this tag to a custom Terrain shader tells Unity that the shader is compatible with the Terrain system, and Unity does not show the warning if a user selects a Material with such shader in **a** ~~the~~ Material Override field."

Reasoning for changes
1. These changes will be used with and without Terrain Toolbox (Removed "using Terrain Toolbox")
2. There is a couple of fields where the user can override/change the material (Changed "the" -> "a")
3. Before this change there wasn't a warning for users (Added "did not")

Reply:
Here's a shorter version:
- Added the TerrainCompatible SubShader Tag. Use this Tag in your custom shader to tell Unity that the shader is compatible with the Terrain system.